.github/workflows/release.yaml

```diff
@@ -109,7 +109,11 @@ jobs:
             ${{ runner.os }}-cargo-release-
 
       - name: Build package
-        run: cargo build --release --target ${{ matrix.target }}
+        run: cargo build --features=snowflake --release --target ${{ matrix.target }}
+
+      - name: Build Linux binary
```

Comment:
Why build twice?

Reply:
On the first run I got this error
`COPY failed: file not found in build context or excluded by .dockerignore: stat target/release/dozer: file does not exist`

Reply:
I think that's because the extra `--target x86_64-unknown-linux-gnu` parameter. We shouldn't build twice.

Reply:
I agree, what's the best solution?

Reply:
Change `ci/Dockerfile` so it finds the build artifact?

Reply:
I removed --target since we're building on the specific architecture

Reply:
I still don't get it. This line https://github.com/getdozer/dozer/pull/1213/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR123 has {{matrix.target}} in the path.